app/src/main/java/com/dsu/pocapplication/news/repository/LoginRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.pocapplication.news.repository
+
+class LoginRepository {
+}
```

Comment:
Loginrepository should not be a part of news package. Please keep the packages news, login inside the package 'repository'

Reply:
done